tox.ini

```diff
@@ -0,0 +1,7 @@
+[tox]
+envlist = py27,py35,py36,py37
+
+[testenv]
+install_command = pip install {opts} {packages} {env:PWD}[tests]
```

Comment:
you want [`extras=tests`](https://tox.readthedocs.io/en/latest/config.html?highlight=extras#conf-extras)